pubsub/subscription.go

```diff
@@ -855,6 +1012,14 @@ func (s *Subscription) updateRequest(cfg *SubscriptionConfigToUpdate) *pb.Update
 		psub.BigqueryConfig = cfg.BigQueryConfig.toProto()
```

Comment:
Why don't we do the same empty-table check for the BQ config?

Reply:
We didn't before and that was what was causing a bug. We fixed it by checking for this properly in the `toProto()` method. I'll make sure CloudStorageConfig is updated in this way as well so we don't have to make these checks here.